code/modules/tgui/states/inventory.dm

```diff
@@ -8,5 +8,8 @@ GLOBAL_DATUM_INIT(inventory_state, /datum/ui_state/inventory_state, new)
 
 /datum/ui_state/inventory_state/can_use_topic(src_object, mob/user)
 	if(!(src_object in user))
+		for(var/obj/item/gripper/G in user.contents) // Because magnetic grippers are snowflakes
+			if(G?.gripped_item == src_object)
```

Comment:
Unsure if the `?.` operator is necessary since `contents` probably won't include `null` values, let alone after filtering to the gripper type.

Reply:
Rewrote the code and I think I solved these issues.